qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -275,6 +255,69 @@ def jobs(
 
         return job_list
 
+    def _build_statuses_filter(self, statuses: List[Union[JobStatus, str]]) -> Dict[str, Any]:
+        """Return the status db filter to use for multiple statuses.
+
+        Returns:
+            The status db filter used to query the api in order to search
+            for jobs based on multiple statuses.
+
+        Raises:
+            IBMQBackendError: If any of the status values is not recognized.
+        """
+        statuses_filter = {'or': []}  # type: Dict[str, List[Dict[str, Any]]]
+        for _status in statuses:
+            _status = self._get_status_filter(_status)
+            statuses_filter['or'].append({'and': [_status]})
```

Comment:
Not sure if it's actually easier, but instead of concatenating a bunch of `and`, you can just do `"inq": [status1, status2...]`. I don't think it works with regular express though (hence might not be easier).

Reply:
Attempted to see if `inq` would support regular expressions and it doesn't seem to be the case. Wasn't able to find much with documentation on whether it is supported either. The main reason I chose to concatenate the statuses together was to maintain the relationship between certain filters. For example:
```
{'status': ApiJobStatus.RUNNING.value, 'infoQueue': {'exists': False}}
```
requires both to be true.